modules/scrabble.js

```diff
@@ -81,7 +81,7 @@ function scrabbleScore (str) {
     .reduce((a, b) => ({
       'score': a['score'] + b['score'],
       'usedBlanks': a['usedBlanks'] + b['usedBlanks']
-    }))
+    }), { 'score': 0, 'usedBlanks': 0 })
```

Comment:
you don't need quotes here
```suggestion
    }), { score: 0, usedBlanks: 0 })
```